src/client/components/autocomplete2/list.tsx

```diff
@@ -57,55 +61,93 @@ export class AutocompleteList extends Component<
     this.setState({ items: newItems })
   }
 
+  onDragEnd = newItems => {
+    const { onDragEnd } = this.props
+    this.setState({ items: newItems })
+    onDragEnd && onDragEnd(newItems)
+  }
+
+  renderListItems = () => {
+    const { formatListItem, isDraggable } = this.props
+    const { items } = this.state
+
+    return items.map((item, i) => {
+      const title = item ? item.title || item.name : ""
+      return (
+        <ListItem key={i} isDraggable={isDraggable}>
```

Comment:
Since `isDraggable` is an optional prop, will `{isDraggable}` result in a false value if one is not provided?

Reply:
Yep exactly, we assume that the component will not be drag and drop unless explicitly specified.